README.rst

```diff
@@ -38,7 +32,14 @@ Additional mailbox functionality.
 Features
 --------
 
-* TODO
+* ``filelock.Filelock`` offers a file locking mechanism that has context-manager support so
+    you can use it in a with statement.
+
+* ``smaildir.Maildir`` is a Gmail-style Maildir mailbox. It uses scandir() instead of listdir(). It also provides a ew iter_folders() method that works like list_folders(), put provides a generator instead of returning a list.
```

Comment:
There's nothing "gmail-style" about this?

Reply:
Done in 3956849b72c52f9ec08cc9240a89e4d3eaad5224